test/ReadsUtils_server_test.py

```diff
@@ -680,6 +687,46 @@ def test_FASTQ_val_fail_bad_ext(self):
                             'File data/sample.txt is not a FASTQ file')
 
     # Upload tests ########################################################
+    def test_upload_fail_min_len_reads(self):
+
+        # The min_Sample.fastq file contains only one read with a length of 1.
+        read_length = 1
+
+        self.fail_upload_reads(
+            {'fwd_file': 'data/min_Sample.fastq',
+             'sequencing_tech': 'seqtech',
+             'wsname': self.ws_info[1],
+             'name': 'filereads1',
+             'min_read_length': read_length + 1},
+            "Invalid FASTQ file - Path: /kb/module/test/data/min_Sample.fastq."
```

Comment:
doesn't the exception give you any info about the problem?

Reply:
no. validateFASTQ returns 0 and `_generate_validation_error_message` creates the error message.

Reply:
Oh, it's from *that* jar. Ok. finding something better has been a TODO for 8 years

Reply:
:+1: